bin/titan/main.go

```diff
@@ -74,16 +75,38 @@ func main() {
 		ListZipThreshold: config.Server.ListZipThreshold,
 	})
 
+	// titan server tls options
+	var servTLSOpts continuous.ServerOption
+	if config.Server.TLSCertFile != "" && config.Server.TLSKeyFile != "" {
+		tlsConfig, err := server.TLSConfig(config.Server.TLSCertFile, config.Server.TLSKeyFile)
+		if err != nil {
+			fmt.Printf("failed to load server TLS config: %s\n", err)
```

Comment:
Zap is recommended for logging.
Fatal contains both fmt and exit operations.
The global logger prints to the normal output by default until there is no redirection.